packages/blocks/src/api/factory.js

```diff
@@ -50,6 +50,10 @@ export function createBlock( name, blockAttributes = {}, innerBlocks = [] ) {
 			result[ key ] = schema.default;
 		}
 
+		if ( schema.source === 'html' && typeof result[ key ] !== 'string' ) {
```

Comment:
This is similar to what was done with `rich-text` source before. It would `create()` here.